packages/block-serialization-default-parser/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@wordpress/block-serialization-default-parser",
+  "version": "1.0.0-rc.0",
+  "description": "Block serialization specification parser for WordPress posts.",
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "wordpress",
+    "block",
+    "parser"
+  ],
+  "homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/block-serialization-default-parser/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WordPress/gutenberg.git"
+  },
+  "bugs": {
+    "url": "https://github.com/WordPress/gutenberg/issues"
+  },
+  "main": "build/index.js",
+  "module": "build-module/index.js",
+  "publishConfig": {
+    "access": "public"
+  }
```

Comment:
I think this needs a babel-runtime dependency cc @gziolo ?

Reply:
I see the following output in the transpiled code:

```js
// ES5
var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

// ESM
import _slicedToArray from "@babel/runtime/helpers/esm/slicedToArray";
```

So it needs to be there.

Reply:
Added in 9463906.